CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+##  v1.12.0 (05/07/2020)
+
+**Common**
+
+* Changed the DiskCache implementation to pass the provided `olp::cache::CacheSettings::enforce_immediate_flush` flag to `leveldb::WriteOptions::sync`.
+* Added a new CMake option `OLP_SDK_ENABLE_DEFAULT_CACHE`. The flag disables the LevelDB and Snappy dependencies and the default cache implementation.
+* Added a new LevelDB environment. The new environment overrides the random access file; by this, we disable mmap system calls, which saves the runtime memory on mobile platforms.
+* Changed the cache `Remove` and `RemoveKeysWithPrefix` API to consider protected keys, i.e., the keys and values are not removed when protected.
+* Fixed a crash during the compaction API call when there is no disk cache present.
+* Fixed the race condition in iOS networking.
+* Added a new `olp::cache::DefaultCache::StorageOpenResult::ProtectedCacheCorrupted` status. The status returned when the protected cache failed to open.
```

Comment:
Status was added but actual returning was reverted so for now it kinda unused

Reply:
Not used now. I think it's worth to mention